include/ecs/entity.h

```diff
@@ -0,0 +1,18 @@
+#ifndef ECS_ENTITY_H
+#define ECS_ENTITY_H
+
+#include <bitset>
+#include "common.h"

```

Comment:
using "ecs/common.h" instead of "common.h" would be more readable imo

Reply:
Are you sure it's not problem with your IDE? Changing common.h -> ecs/common.h works good for me.

Reply:
Seems like an IDE problem, but I'm gonna leave common.h for people using JetBrains :d